client/asset/dcr/dcr.go

```diff
@@ -65,7 +65,7 @@ const (
 )
 
 var (
-	requiredWalletVersion = dex.Semver{Major: 8, Minor: 4, Patch: 0}
+	requiredWalletVersion = dex.Semver{Major: 8, Minor: 5, Patch: 0}
```

Comment:
Requires unreleased dcrwallet (unmerged work actually).  We technically already have a bug if the user decided to use individually-encrypted accounts, and fixing it properly would seem to require determining if the account is encrypted, otherwise we might be failing (un)lock appropriately.